modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -308,20 +313,22 @@ public function createReservation(AlephPatron $patron, AlephRequest $request) {
 
     $options['body'] = 'post_xml=' . $xml->asXML();
 
-    // BIB library code + the system number.
-    // For example, USM01000050362.
     $rid = $this->mainLibrary . $request->getDocNumber();
 
-    // ADM library code + the item record key.
-    // For example, USM50000238843000320.
-    $iid = $request->getInstitutionCode() . $request->getDocNumber() .
-      $request->getItemSequence();
+    foreach ($holding_groups as $url => $holding_group) {
+      $response = $this->requestRest(
```

Comment:
How are errors handled here? RuntimeExceptions? Seems a bit unreliable.
Would be nicer if the function handled the errors and had a clear documented behaviour when errors orccured such as returning NULL or a business-exception.